pagopa/io-app-design-system
# chore: add `switchCard` icon

## Short description
Added the SwitchCard icon

## List of changes proposed in this pull request
- Added `IconSwitchCard` component
- Added `switchCard` icon to `IOIcons`

## How to test
In the example app, go to the Icons section and check that the icon is correctly rendered
